jujugui/static/gui/src/app/components/user-profile/user-profile.js

```diff
@@ -55,6 +56,24 @@ YUI.add('user-profile', function() {
     },
 
     componentWillMount: function() {
+      if (this.props.authenticated) {
```

Comment:
This is causing the 'no data' splash page to be shown even if you have a sandbox model. Which means once you're in the profile page you cannot get out without logging into charmstore.